prepare.js

```diff
@@ -4,22 +4,14 @@ const libPackage = require('./package.json');
 // remove not required fields
 delete libPackage.devDependencies;
 
-// use only required temporary script in dist
-libPackage.scripts = {
-  postversion: 'cd .. && node bump.js',
-};
-
 libPackage.publishConfig = {
   access: 'public',
 }
 
 // include all 'dist/*' files, but bundles
 libPackage.files = ['*', '!*.bundle.js'];
 
-fs.mkdirSync('./dist', { recursive: true });
-fs.copyFileSync('README.md', './dist/README.md');
-fs.copyFileSync('LICENSE', './dist/LICENSE');
 fs.writeFileSync(
-  './dist/package.json',
```

Comment:
Since the new generator correctly uses the `files` array, we no longer need to build out this `dist` and associated package.json. I left in the removal of `devDependencies` and setting the `publishConfig` to public.